composer.json

```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": "^7.4"
+        "php": ">=7.4"
```

Comment:
I would rather set it to `^7.4 || 8.0.*`

Reply:
It looks like version with `||` is more widespread:

https://github.com/symfony/symfony/blob/5.x/composer.json#L19

https://github.com/symfony/symfony/blob/5.x/composer.json#L19
https://github.com/thephpleague/oauth2-client/blob/master/composer.json#L9
https://github.com/laravel/laravel/blob/8.x/composer.json#L11

But maybe `^7.4 || ^8.0` would be better?

Reply:
Hmm... there shouldn't be any breaking changes in 8.1, so I'm ok with that.

Reply:
ok, changed ;)